### ISSUE TITLE
Unable to load `lena.png` by `imageio` correctly, different  with `cv2` or `PIL` or `skimage`.

### ISSUE THREAD

**Issue by ausk** [imageio/imageio#369]
Dear guys, I'm trying to read image in different library. When I use `cv2` or `PIL` or `skimage` to read `lena.png`, they are all the same. But then I read it using `imageio`, the image data become different.

The source image:  [lena.png](https://i.stack.imgur.com/p0C3w.png)

The result: [result](https://i.stack.imgur.com/vWprM.png)

The test code:
```python
#!/usr/bin/python
# 2018.08.19 14:17 (+0800)

import cv2 
import PIL
import imageio 
import skimage.io  
import matplotlib.pyplot as plt 
import numpy as np 

#You can get `lena.png` from url:  https://i.stack.imgur.com/p0C3w.png
fname = "lena.png" 

## Read 
a = np.array(imageio.imread(fname))
b = cv2.imread(fname)[...,::-1]
c = np.array(PIL.Image.open(fname))
d = skimage.io.imread(fname)

# Check in value 
print( (a==b).all() )  # False 
print( (b==c).all() and (c==d).all()) # True 

## Display 
imgs = [a, b, c, d]
titles = ["imageio", "PIL", "cv2", "skimage"]
for i, (img, title) in enumerate(zip(imgs, titles), start=1):
    plt.subplot(2, 2, i)
    plt.imshow(img)
    plt.title(title)
    plt.axis("Off")

plt.show()

## Version
print(imageio.__version__) # 2.3.0

```

**Comment by ArgentumWalker** [comment]
Have same problem when trying open image with PyPlot (imread). It seems like proble only with red and green channels (I've looked at diff between ImageIO and PyPlot results).

**Comment by almarklein** [comment]
This looks like a duplicate of #366. Could you please check whether this would fix it?
```py
imageio.imread(fname, ignoregamma=True)
````

**Comment by almarklein** [comment]
Closed by #374, please re-open if the assertion in my previous comment is False.
